_community/utopianio.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Utopian.io 
+position: 3
+---
+
+Utopian Platform rewards everyone who contributes to open-source projects. The community includes free software enthusiast, moderators and supervisors who look after quality of contributions being posted by developers.
+
+**Official webpage** - [https://utopian.io/](https://utopian.io/)
+
+**Official NPM module** - [https://github.com/utopian-io/npm/](https://github.com/utopian-io/npm)
```

Comment:
Can you please add an explanation for what the "Official NPM module" link is about

Reply:
Updated.